dps-gradle-spring-boot-suppressions.xml

```diff
@@ -54,4 +54,131 @@
     </packageUrl>
     <cve>CVE-2018-1258</cve>
   </suppress>
-</suppressions>
\ No newline at end of file
+
+  <suppress>
```

Comment:
Shouldn't this be added to .ignore? This is the one generated by gradle at compile time right?

Reply:
removed this